### ISSUE TITLE
Filter to search books by skill teached (for V or e menus. or both)

### ISSUE THREAD

**Issue by adamkad1** [CleverRaven/Cataclysm-DDA#47530]
# Is your feature request related to a problem? Please describe.
Finding the exact book you need in a large pile (for npc teaching or sth) is hard as hell if you dont put notes on all your books, so i wish we could search books by skill

# Describe the solution you'd like
something like b:devices which will show only books that teach devices skill

# Describe alternatives you've considered
idk

# Additional context
idk

**Comment by jagoly** [comment]
I agree this might be useful, but for now, why not just use the (R)ead menu to search for skill books? Find the book you want that way, then in the (G)rab menu search for it by name.

Also as a side note, I think a new loot zone category should be added just for skill books, similar to the Food/P.Food split. Not sure if there's already an issue for that.

**Comment by BrettDong** [comment]
Implemented in #48275.
